initfiles/etc/DIR_NAME/config2mgr/esp.xml.in

```diff
@@ -18,15 +18,11 @@
 -->
 <Environment>
   <Software>
-    <EspProcess build="${CPACK_RPM_PACKAGE_VERSION}_${CPACK_RPM_PACKAGE_RELEASE}" description="ESP server" componentfilesDir="${COMPONENTFILES_PATH}" enableSEHMapping="true" enableSNMP="false" formOptionsAccess="false" httpConfigAccess="true" logDir="${LOG_PATH}/config2mgr" logLevel="10" logRequests="true" logResponses="true" txSummaryLevel="1" txSummaryResourceReq="false" maxBacklogQueueSize="200" maxConcurrentThreads="0" name="config2mgr" perfReportDelay="60" computer="localhost" directory="${PID_PATH}/config2mgr">
+    <EspProcess build="${CPACK_RPM_PACKAGE_VERSION}_${CPACK_RPM_PACKAGE_RELEASE}" description="ESP server" componentfilesDir="${COMPONENTFILES_PATH}" enableSEHMapping="true" enableSNMP="false" formOptionsAccess="false" httpConfigAccess="true" logDir="${LOG_PATH}/config2mgr" logLevel="10" logRequests="true" logResponses="true" txSummaryLevel="1" txSummaryResourceReq="false" maxBacklogQueueSize="200" maxConcurrentThreads="0" name="configmgr" perfReportDelay="60" computer="localhost" directory="${PID_PATH}/config2mgr">
```

Comment:
should 'directory="${PID_PATH}/config2mgr"' changed to 'directory="${PID_PATH}/configmgr"' for consistence?

Reply:
There is already a configmgr for the existing config manager. I am leaving both in place now, that way you explicitly run the new one. When the new one replaces the old one, it will replace it in the configmgr directory, that way users won't have to change how they run it. They will just get the new one.